### ISSUE TITLE
Requirement of GitHub actions and Templates

### ISSUE THREAD

**Issue by AbhijithGanesh** [CodXCrypt/Whatsapp-API2#26]
We need pull-request templates, issue templates and github actions for this project. I would like to take the responsibility for the templates initially and later If no one takes responsibility for actions, I shall work on it too.

**Comment by AbhijithGanesh** [comment]
great, please do if it can be done.

**Comment by unnati914** [comment]
uhm if it is marked for hactoberfest then it can only be solved in that period ig

**Comment by AbhijithGanesh** [comment]
I shall wait till October, if it is marked for Hacktober. I would like to be assigned though.

**Comment by unnati914** [comment]
assigned

**Comment by rohan-kulkarni-25** [comment]
I think we should get it done because we would need these things during hacktober fest !! WE will need Templates for our Project !

**Comment by AbhijithGanesh** [comment]
@rohan-kulkarni-25 I agree but I will get this done on first of october. First thing on October. Every little thing counts I suppose. If you insist , I wouldn't mind sending in the templates immediately.

**Comment by rohan-kulkarni-25** [comment]
> @rohan-kulkarni-25 I agree but I will get this done on first of october. First thing on October. Every little thing counts I suppose. If you insist , I wouldn't mind sending in the templates immediately.

As PA says
 
CC @himanshu007-creator  @unnati914

**Comment by himanshu007-creator** [comment]
Hi @AbhijithGanesh, if it's regarding Hacktoberfest, it's completely Okay to wait. But if it's not your first time, and you feel you can still find other issues to contribute somewhere else, you are requested to start the work A.S.A.P 🤖

**Comment by AbhijithGanesh** [comment]
I am working on this, it is important and feels like need of the hour.

**Comment by rohan-kulkarni-25** [comment]
These templates are in form of yml ? How will these work ? @AbhijithGanesh

**Comment by AbhijithGanesh** [comment]
Templates which are yml will be implemented as forms, 
GitHub Actions will be internally running tests for us if we configure yml well

**Comment by rohan-kulkarni-25** [comment]
> Templates which are yml will be implemented as forms,
> GitHub Actions will be internally running tests for us if we configure yml well

Yeah I thought same but I think it's not the case

**Comment by rohan-kulkarni-25** [comment]
No issue template is created and actions also don't run

**Comment by AbhijithGanesh** [comment]
I think it is a directory issue, I will patch it immediately.
